open-craft/labxchange-xblocks
# Removing transcript support from student_view_data

**Description**

- [FAL-2483](https://tasks.opencraft.com/browse/FAL-2483)
- [LX-2243](https://tasks.opencraft.com/browse/LX-2243)

As a part of LX-2243, we are removing the transcript view for the audio xblock.
Transcript will no longer be shown in the xblock.

**Testing Instructions**
1. Add the labxchange xblock into your labxchange devstack.
2. Add a new audio along with a transcript and save the video
3. Check that the transcript are not rendered by the iframe.
